addons/patches/optre/functions/fn_PelicanLoadValidate.sqf

```diff
@@ -31,6 +31,10 @@ switch true do {
         _veh attachTo [_pelican, [0,-7.2, -0.37105]];
         _veh setDir 180;
     };
+    case (_veh isKindOf "Car_F") : {
```

Comment:
Take a peek at the sample I wrote on the [prior comment](https://github.com/WitchFreya/SWSAddons/pull/8#discussion_r1397672390); since these numbers are the same you can include them in the case for Tank_F above.

Reply:
Gonna merge this and just modify it myself since I wanna get an update out before op today.